cfgov/jinja2/v1/_includes/organisms/sidebar-breakout.html

```diff
@@ -41,72 +44,46 @@
                           block
                           block__flush-top
                           block__flush-bottom">
-                {{ _sidebar(settings) }}
+              {% for block in page.sidebar_breakout %}
+                {% if 'heading' in block.block_type %}
+                  <h3 class="o-sidebar-breakout_heading">
+                      {{ block }}
+                  </h3>
+                {% elif 'breakout_image' in block.block_type %}
+                  {% set blocks = block.value.bound_blocks %}
+                  {% if blocks.image is defined %}
+                    {% set photo = image(blocks.image.value, 'original') %}
+                    <div class="o-sidebar-breakout_img-container
+                                {{ 'o-sidebar-breakout_img-container__round'
```

Comment:
Why not simplify our code and let designers create round images? One less complexity that we then have to manage.